app/src/main/kotlin/compiler/lowlevel/storage/DisplayStorage.kt

```diff
@@ -1,10 +1,12 @@
 package compiler.lowlevel.storage
 
-import compiler.ast.Program
 import java.io.PrintWriter
 
-class DisplayStorage(val program: Program) {
+class DisplayStorage(private val programStaticDepth: Int) {
+
+    val displayLabel: String = "display"
```

Comment:
This label should not be defined in two places (here and in `DefaultFunctionDetailsGenerator.kt`).

Reply:
Didn't see it. I'm gonna remove it from details generator.

Reply:
Actually, the other way around wold be preferable, so that `intermediate` packages does not depend on `lowlevel`.

Reply:
ok ok

Reply:
it was more intuitive to me this way

Reply:
I guess the direction of dependencies between packages in this project (low-level to high-level or the other way around) is debatable, but let's keep it as it is for now.